spec/controllers/enterprises_controller_spec.rb

```diff
@@ -139,4 +139,18 @@
       expect(response.status).to be 409
     end
   end
+
+  context "checking access on nonexistent enterprise" do
+    before do
+      spree_get :shop, id: "some_nonexistent_enterprise"
+    end
+
+    it "redirects to shops_path" do
+      expect(response).to redirect_to shops_path
+    end
+
+    it "redirects to shops_path" do
+      expect(request.flash[:error]).to eq(I18n.t(:enterprise_shop_show_error))
```

Comment:
you can move this expect to the previous test or give this test a better name related to showing an error

Reply:
Sorry. Cmd C + Cmd V problems hahaha.

Reply:
All good now.